tests/unit_tests/test_optional_enhancements.py

```diff
@@ -0,0 +1,43 @@
+import pytest
+from swap_meet.item import Item
+from swap_meet.vendor import Vendor
+from swap_meet.clothing import Clothing
+from swap_meet.decor import Decor
+from swap_meet.electronics import Electronics
+
+#@pytest.mark.skip
```

Comment:
Really nice work the optional enhancements y'all! The test are well written as well! Very proud of the work you all turned in!